backend/apps/core/api/status.py

```
@@ -0,0 +1,22 @@
+    Return the current application version.
+    
+    Args:
+        request: HTTP request object (unused but required by Django)
+        
+    Returns:
+        JsonResponse containing the application version
+    """
+    if request.method != 'GET':
+        return JsonResponse(
+            {'error': f'Method {request.method} is not allowed'},
+            status=requests.codes.method_not_allowed
```

Ruff (0.11.9)

Undefined name `requests`

(F821)